ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -280,6 +293,49 @@ - (void)testShouldHandleVnodeOpEvent {
     XCTAssertEqual(kauthResult, KAUTH_RESULT_DEFER);
 }
 
+- (void)testCurrentProcessWasSpawnedByRegularUser {
+    // Defaults should pass for all tests
+    XCTAssertTrue(CurrentProcessWasSpawnedByRegularUser());
+    MockProcess_Reset();
+
+    // Process tree with all UIDs less than 500 were not spawned by regular users
```

Comment:
Should we have more than one process in this tree?

Reply:
I'll update the comment.  This should test for process wiht Id under 500, not having a parent